examples/custom.py

```diff
@@ -1,11 +1,16 @@
+from typing import TYPE_CHECKING
+
 from astroid import nodes
 
 from pylint.checkers import BaseChecker
 from pylint.interfaces import IAstroidChecker
 
+if TYPE_CHECKING:
+    from pylint.lint import PyLinter
 
-# This is our checker class.
```

Comment:
Why remove this?

Reply:
The class is called `` MyAstroidChecker`` so I thought it was self explanatory without the comment.

Reply:
Hm yeah. Since it is "documentation" it doesn't hurt to be as explicit as possible, but I don't have a preference.